python/neuroglancer/pipeline/scripts/queue_pickle.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+
+"""
+Save or load a cloud queue to or from a json file.
+
+Example:
+  python queue_pickle.py save pull-queue --file my-job.json
+
+"""
+
+from datetime import datetime
+import json
+import math
+
+import click
+from tqdm import tqdm
+
+from neuroglancer.pipeline import TaskQueue
+
+def deserialize(file):
+  with open(file, 'r') as f:
+    queue = json.loads(f.read())
+  return queue
+
+@click.command()
+@click.argument('queue_name')
+@click.option('--file', default=None, help='Savefile to restart from.')
+def save(queue_name, file):
+  click.echo("Remember to ensure the queue is not running and no leases are active!")
+
+  queue = []
+
+  if file is None:
+    timestamp = datetime.today().strftime("%Y-%m-%d-%H:%M")
+    file = './{}-{}.json'.format(queue_name, timestamp)
+  else:
+    queue = deserialize(file)
+    print "Restarting with {} as seed. Tasks: {}".format(file, len(queue))
```

Comment:
wait there isn't any restarting here right? We're only saving. so wouldn't the message be more appropriate as something along the lines of ```Using existing seed file {} .... ```